scripts/ci/run-test-msbuild.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+OLD_CWD=`pwd`
+TESTCMD=`realpath ${TESTCMD}`
+
+${TESTCMD} --label=remove-mono-temp-dir --timeout=1m rm -rf /tmp/xplat-master
+${TESTCMD} --label=autogen --timeout=15m ./autogen.sh --prefix=/tmp/mono-from-source
+${TESTCMD} --label=make --timeout=30m make PROFILE=net_4_x -j 2
+${TESTCMD} --label=make-install --timeout=15m make install
+
+OLD_PATH=${PATH}
+export PATH=/tmp/mono-from-source/bin/:${OLD_PATH}
+${TESTCMD} --label=cert-sync --timeout=15m cert-sync /etc/ssl/certs/ca-certificates.crt
+
+${TESTCMD} --label=remove-msbuild-temp-dir --timeout=1m rm -rf /tmp/xplat-master
+${TESTCMD} --label=clone-msbuild --timeout=5m git clone https://github.com/mono/msbuild.git /tmp/xplat-master
```

Comment:
I wonder if we should grep the msbuild commit out of `packaging/MacSDK/msbuild.py` instead of always taking xplat-master?

Reply:
That seems like a good idea. We always want to be using the same msbuild on mac and linux, right?

Reply:
They ought to always match. When they don't, it's usually because nobody told me to update Linux.

Reply:
It sounds like the right fix is to verify that the macsdk hash matches xplat-master head or matches linux-packaging?